homework/index.js

```diff
@@ -30,17 +32,134 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, repositories) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+  function renderError(error) {
+    const container = document.getElementById('container');
+    container.innerHTML = '';
+    createAndAppend('div', container, {
+      text: error.message,
+      class: 'error',
```

Comment:
The assignment asked for you to use the class `alert-error` (step 6 of the functional requirements https://github.com/HackYourFuture/JavaScript3/pull/296/files#r307708657)

My unit tests that I use to help me review the homework expects this. In general, you should meticulously follow the functional requirements. It is what the customer (in this case, me) expects.